website/content/docs/commands/cache/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: cache - Command
-description: |-
-  Boundary's "cache" command lets you manage a local cache of information to facilitate searches and protect your system resources from being overwhelmed.
+description: >-
+  Boundary's "cache" command performs operations on a local cache that facilitates searches and protects your system resources from being overwhelmed.
```

Comment:
I would change this to use the format of the other descriptions by removing "Boundary's ..."

```suggestion
  The "cache" command performs operations on a local cache that facilitates searches and protects your system resources from being overwhelmed.
```